avaframe/ana3AIMEC/ana3AIMEC.py

```diff
@@ -0,0 +1,1149 @@
+import sys
+import os
+import logging
+import glob
+import math
+import numpy as np
+import scipy as sp
+import pandas as pd
+import copy
+import functools
+import operator
+from multiprocessing import Pool
+from matplotlib import pyplot as plt
+import matplotlib
+import matplotlib.lines as mlines
+import matplotlib.patches as mpatches
+from matplotlib import cm
+
+# Local imports
+import avaframe.in2Trans.shpConversion as shpConv
+import avaframe.in2Trans.geoTrans as geoTrans
+import avaframe.in3Utils.ascUtils as IOf
+
+# create local logger
+log = logging.getLogger(__name__)
+
+# -----------------------------------------------------------
+# Aimec read inputs tools
+# -----------------------------------------------------------
+
+
+def readAIMECinputs(avalancheDir):
+    """
+    Reads the requiered files for AIMEC postpocessing
+    given an avalanche directory
+    """
+    cfgPath = {}
+    pathPressure = avalancheDir + '/Outputs/com1DFA/dfa_pressure'
+    pathFlowHeight = avalancheDir + '/Outputs/com1DFA/dfa_depth'
+    pathMassBalance = avalancheDir + '/Outputs/com1DFA/dfa_mass_balance'
+
+    if not os.path.exists(pathMassBalance):
+        os.makedirs(pathMassBalance)
+
+    profileLayer = glob.glob(avalancheDir + '/Inputs/LINES/*aimec*.shp')
+    cfgPath['profileLayer'] = ''.join(profileLayer)
+
+    demSource = glob.glob(avalancheDir + '/Inputs/*.asc')
+    try:
+        assert len(demSource) == 1, 'There should be only one and only one DEM .asc file in ' + \
+            avalancheDir + '/Inputs/'
+    except AssertionError as e:
+        raise
+    cfgPath['demSource'] = ''.join(demSource)
+    pressurefileList = [pathPressure +
+                        '/' +
+                        str(name) for name in
+                        sorted(os.listdir(pathPressure)) if os.path.isfile(os.path.join(pathPressure, name))]
+    cfgPath['pressurefileList'] = pressurefileList
+
+    depthfileList = [str(pathFlowHeight) +
+                     '/' +
+                     str(name) for name in
+                     sorted(os.listdir(pathFlowHeight)) if os.path.isfile(os.path.join(pathFlowHeight, name))]
+    cfgPath['depthfileList'] = depthfileList
+
+    massfileList = [str(pathMassBalance) +
+                    '/' +
+                    str(name) for name in
+                    sorted(os.listdir(pathMassBalance)) if os.path.isfile(os.path.join(pathMassBalance, name))]
```

Comment:
Maybe split this for readability? Or break out into function, as it does the same thing 3 times?

And use os.path.join wit os.pathsep  (to make it platform independent....)